features/content_crop.feature

```diff
@@ -193,6 +189,59 @@ Feature: Cropping the Image Articles
 
     @auth
     @vocabulary
+    Scenario: Create a story with a featured image
+      When upload a file "bike.jpg" to "archive" with "bike"
+      And we post to "/archive"
+      """
+      [{"guid": "feature_image", "slugline": "feature_image"}]
+      """
+      When we patch "/archive/feature_image"
+      """
+      {
+      	"associations": {
+      		"feature_image": {
+      			"_id": "bike",
+      			"poi": {"x": 0.2, "y": 0.3}
+      		}
+      	}
+      }
+      """
+      Then we get OK response
+      And we get existing resource
+      """
+      {
+      	"associations": {
+      		"feature_image": {
+      			"_id": "bike",
+      			"poi": {"x": 0.2, "y": 0.3},
+      			"renditions": {
+      				"16-9" : {
+      					"poi" : {"y" : 216, "x" : 108}
```

Comment:
poi is in pixel in rendition, but it's a proportion in the parent object. It that on purpose ?